packages/react-core/src/components/Page/__tests__/Generated/__snapshots__/Page.test.tsx.snap

```diff
@@ -5,7 +5,7 @@ exports[`Page should match snapshot (auto-generated) 1`] = `
   value={
     Object {
       "isManagedSidebar": false,
-      "isNavOpen": false,
+      "isNavOpen": true,
```

Comment:
did something else happen that i missed

Reply:
The first `handleResize` was debounced 250ms. I've fixed it.